src/ServiceControl.Plugin.CustomChecks.Sample/EndpointConfig.cs

```diff
@@ -1,19 +1,19 @@
 
 namespace ServiceControl.Plugin.CustomChecks.Sample
 {
-    using NServiceBus;
-    
+	using NServiceBus;
+	
 
-    /*
+	/*
 		This class configures this endpoint as a Server. More information about how to configure the NServiceBus host
 		can be found here: http://particular.net/articles/the-nservicebus-host
 	*/
-    public class EndpointConfig : IConfigureThisEndpoint, AsA_Server
```

Comment:
I raised that question before https://github.com/Particular/ServiceControl.Plugin.Nsb6.CustomChecks/pull/2#issuecomment-176049412